internal/checker/cloudformation/rules/logging/verify_s3_bucket_read_permissions_restricted.rego

```diff
@@ -0,0 +1,48 @@
+# METADATA
+# title: "Verify S3 Bucket Restricts READ Permissions from Everyone"
+# description: "This policy verifies that the specified S3 bucket is configured to restrict READ permissions from everyone. By enforcing this restriction, the policy aims to mitigate the risk of unauthorized access to sensitive data stored within the bucket, thus enhancing overall security."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-s3-bucket.html
+# custom:
+#   id: CB_CFT_011
+#   severity: CRITICAL
+package lib.cloudformation.CB_CFT_011
+import future.keywords.in
+
+resource := "AWS::S3::Bucket"
+
+is_valid {
+	some resources in input.Resources
+	resources.Type == resource
+}
+
+pass[resources] {
```

Comment:
update implementation to

pass[read_permissions] {
 	is_valid
    some resources in input.Resources
    read_permissions := resources.Properties
    forbidden_values := ["PublicReadWrite", "PublicRead"]
    not read_permissions.AccessControl in forbidden_values
}